app/models/teams/account.rb

```diff
@@ -18,22 +18,12 @@ class Teams::Account < ActiveRecord::Base
   has_many :plans, through: :account_plans
   belongs_to :admin, class_name: 'User'
 
-  validates :team_id, presence: true, uniqueness: true
   validates_presence_of :stripe_card_token
   validates_presence_of :stripe_customer_token
+  validates :team_id, presence: true, uniqueness: true
 
   attr_protected :stripe_customer_token, :admin_id
 
-  validate :stripe_customer_token, presence: true
```

Comment:
what about 
validate :stripe_customer_token, presence: true , if: ->(r){r.state == 'active'}
we need a migration to add add this field :string, default: 'pending'